Gopkg.toml

```diff
@@ -60,3 +60,7 @@
 [[constraint]]
   revision = "v1.3.1"
   name = "github.com/golang/protobuf"
+
+[[constraint]]
+  name = "github.com/ghodss/yaml"
```

Comment:
Instead of adding another yaml package, we could likely use the already existing one built into k8s that supports both JSON and YAML. It's worked well for me in the past and would ensure that we're using the same parser/decoder as k8s itself: https://godoc.org/k8s.io/apimachinery/pkg/util/yaml

Reply:
Hmm...from what I can tell that library doesn't seem to have a Marshaler

Reply:
Fixed per discussion offline